packages/react/src/components/Tooltip/DefinitionTooltip.tsx

```diff
@@ -104,6 +104,9 @@ const DefinitionTooltip: React.FC<DefinitionTooltipProps> = ({
       onMouseEnter={() => {
         openOnHover ? setOpen(true) : null;
       }}
+      onFocus={() => {
+        openOnHover ? setOpen(true) : null;
```

Comment:
Should this open on focus regardless of the `openOnHover` prop? Reading over the issue it seems like no matter what, it should open on focus.